packages/astro/src/@types/astro.ts

```diff
@@ -1473,15 +1473,15 @@ export interface AstroUserConfig {
 			 * @docs
 			 * @kind h4
 			 * @name experimental.i18n.locales
-			 * @type {string[]}
+			 * @type {Locales}
 			 * @version 3.5.0
 			 * @description
 			 *
-			 * A list of all locales supported by the website (e.g. `['en', 'es', 'pt-br']`). This list should also include the `defaultLocale`. This is a required field.
+			 * A list of all locales supported by the website, include the `defaultLocale`. This is a required field.
```

Comment:
```suggestion
			 * A list of all locales supported by the website, including the `defaultLocale`. This is a required field.
```
Oops, I made a typo.